data/autoyast_qam/12-common_base_installation.xml.ep

```diff
@@ -389,6 +394,9 @@
     <pesize/>
     <type config:type="symbol">CT_DISK</type>
     % }
+    % if ($check_var->('MULTIPATH', '1')) {
+    <device>/dev/mapper/0QEMU_QEMU_HARDDISK_hd0</device>
+    % }
```

Comment:
One minor comment here,  if we don't use these hardcode `/dev/mapper/0QEMU_QEMU_HARDDISK_hd0`, can we pass the job? ：）

Reply:
Not on SLE 12 https://dzedro.suse.cz/tests/819,
but I removed it on SLE 15, had issue on 15-SP2 which was at the end something else and left it there, but it's not needed.